core/scripts/vrfv2plus/testnet/proofs.go

```diff
@@ -154,6 +154,5 @@ func generateProofForV2Plus(e helpers.Environment) {
 		rc.CallbackGasLimit,
 		rc.NumWords,
 		rc.Sender,
-		rc.NativePayment,
```

Comment:
Lets print the `ExtraArgs` hex-encoded using `hexutil`

Reply:
Done